pkg/client/manager_client.go

```diff
@@ -6,6 +6,11 @@ import (
 	"github.com/golang/protobuf/ptypes/empty"
 	"golang.org/x/net/context"
 	"google.golang.org/grpc"
+<<<<<<< HEAD
+=======
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/protobuf/types/known/emptypb"
+>>>>>>> c6cc6d2 (refactor: replace deprecated grpc.WithInsecure)
```

Comment:
@c3y1huang fix this.

Reply:
https://github.com/longhorn/backing-image-manager/pull/157#issuecomment-1859564826